### PR TITLE
Tvp tests

### PR DESCRIPTION
**Pull request by NikitaDef**
Changed some tests.
Changed makeParamExtra `case TVP`

### CODE REVIEW

**Comment by NikitaDef** [comment]
@v-kaywon will you merge it?

File: tvp_go19.go [hunk 1]
```diff
@@ -58,17 +59,16 @@ func (tvp TVP) check() error {
...
 
-func (tvp TVP) encode(schema, name string) ([]byte, error) {
-	columnStr, tvpFieldIndexes, err := tvp.columnTypes()
-	if err != nil {
-		return nil, err
+func (tvp TVP) encode(schema, name string, columnStr []columnStruct, tvpFieldIndexes []int) ([]byte, error) {
+	if len(columnStr) != len(tvpFieldIndexes) {
+		return nil, ErrorWrongTyping
 	}
 	if len(columnStr) != len(tvpFieldIndexes) {
```
Comment (yukiwongky) [inline tvp_go19.go]: Can you remove this check? It's the same check as above.

**Review by yukiwongky — CHANGES_REQUESTED** [review]

**Comment by yukiwongky** [comment]
@NikitaDef  Can you make the minor change mentioned above? Other than that I think everything looks good.

File: tvp_go19.go (hunk 1 above)
Comment (NikitaDef) [reply tvp_go19.go]: removed

**Comment by NikitaDef** [comment]
@v-kaywon I made change

**Pull request merged**